cpp/oneapi/dal/algo/decision_forest/backend/gpu/infer_train_cls_kernel_dpc_test.cpp

```diff
@@ -53,7 +53,10 @@ TEST(infer_and_train_cls_kernels_test, can_process_simple_case_default_params) {
     const auto y_train_table =
         dal::homogen_table{ queue, y_train, row_count_train, 1, dal::empty_delete<const float>() };
 
-    const auto x_test_table = dal::homogen_table{ x_test_host,
+    auto x_test = sycl::malloc_shared<float>(row_count_test * column_count, queue);
+    queue.memcpy(x_test, x_test_host, sizeof(float) * row_count_test * column_count).wait();
```

Comment:
If I am right, `queue.memcpy()` shall be used only with USM pointers. In that case to copy data from host pointer to the USM one, `std::memcpy()` shall be used.

Reply:
fixed, thanks